pkg/expression/builtin.go

```diff
@@ -725,6 +725,7 @@ var funcs = map[string]functionClass{
 	ast.SecToTime:        &secToTimeFunctionClass{baseFunctionClass{ast.SecToTime, 1, 1}},
 	ast.Second:           &secondFunctionClass{baseFunctionClass{ast.Second, 1, 1}},
 	ast.StrToDate:        &strToDateFunctionClass{baseFunctionClass{ast.StrToDate, 2, 2}},
+	ast.ToDate:           &toDateFunctionClass{baseFunctionClass{ast.ToDate, 1, 2}},
```

Comment:
How about also be kept consistent with what @joechenrh mentioned when we create new functions in `expression` package🤔

Reply:
> How about also be kept consistent with what @joechenrh mentioned when we create new functions in `expression` package🤔

Maybe we could put ast.XXX(function name) together, but functionClass has own place.